.github/workflows/ci.yaml

```diff
@@ -265,6 +280,9 @@ jobs:
         with:
           command: test
           args: --offline --workspace
+        env:
+          DATABASE_URL: postgresql://postgres:postgres@localhost/postgres
+          SQLX_OFFLINE: '1'
```

Comment:
This is needed, else sqlx will try to type check the queries against the live database